garak/resources/common.py

```diff
@@ -43,19 +45,25 @@
 
 
 def load_advbench(size: int = 0) -> pd.DataFrame:
-    advbench_base_path = (
-        garak._config.transient.package_dir
-        / "resources"
-        / "advbench"
-        / "harmful_behaviors.csv"
-    )
+    try:
+        advbench_base_path = data_path / "advbench" / "harmful_behaviors.csv"
+    except GarakException:
+        logging.info(
+            "No default harmful_behaviors.csv found download or cached file will be utilized."
+        )
```

Comment:
This provides an example for a case where the tooling would allow override of a resource not shipped with `garak` by placing a file in the `data` path using a known location.